roles/servicetelemetry/defaults/main.yml

```diff
@@ -74,6 +74,7 @@ servicetelemetry_defaults:
       deployment_size: 1
       web:
         enabled: false
+      auth: basic
```

Comment:
Changing this to be the default means that documentation is mandatory before we release